README.md

```diff
@@ -171,7 +171,7 @@ is bootstrapped, open PowerShell and run:
 
 ``` powershell
 PS:\> .\vcpkg integrate install
-PS:\> .\vcpkg install --triplet x64-windows libpng sdl2 sdl2-net opusfile fluidsynth
+PS:\> .\vcpkg install --triplet x64-windows libpng sdl2 sdl2-net opusfile fluidsynth pdcurses
```

Comment:
Do not place this dependency in README.md, please. It's not enough, and describing all the steps needed to enable the feature will complicate README.md too much.

If we want to include this information, then it's place is in `docs/build-windows.md`. Probably in a new section, as enabling the debugger feature requires additionally manual edit on `config.h` file…

Reply:
adding pdcurses to the package - reduces the afford down to config.h/#define C_DEBUG 1 - or what else?

Reply:
Yes, user needs to edit `C_DEBUG` and `C_HEAVY_DEBUG` as they wish and remember to select "Release" build option in Visual Studio.

Also, this information will be made obsolete once we'll replace Visual Studio solution with VS+Meson, so after that these documentation changes will need to be removed anyway.

Reply:
All good info - rolled it into the update.